pulpcore/cli/ansible/distribution.py

```diff
@@ -79,6 +80,7 @@ def distribution(ctx: click.Context, pulp_ctx: PulpContext, distribution_type: s
 distribution.add_command(show_command(decorators=lookup_options))
 distribution.add_command(destroy_command(decorators=lookup_options))
 distribution.add_command(create_command(decorators=create_options))
+distribution.add_command(label_command())
```

Comment:
This is added in an upcoming pulp_ansible version, right?
Can we pass a list of needed plugins with versions to the `label_command` generator?
And than we should have a test that can excercise this at least on the nightly container.